playbooks/site.yml

```diff
@@ -120,6 +125,23 @@
   roles:
     - geerlingguy.mysql
     - chaos_jetzt.freescout
+  tasks:
+    - name: clone the httpheaderauth-module
+      git:
+        repo: 'https://github.com/chaos-jetzt/freescout-module-httpheaderauth.git'
+        dest: "{{ freescout_basedir }}/Modules/HttpHeaderAuth"
+    - name: enable the HttpHeaderAuth module
+      become: yes
+      become_user: "{{ freescout_user }}"
+      command:
+        cmd: "php artisan module:enable HttpHeaderAuth"
+        chdir: "{{ freescout_basedir }}"
+    - name: migrate the HttpHeaderAuth module
+      become: yes
+      become_user: "{{ freescout_user }}"
+      command:
+        cmd: "php artisan module:migrate HttpHeaderAuth -n --force"
```

Comment:
Here also a question: Is the force option needed? For me it seems ugly to use `force` when not needed.

Reply:
Yes, it is needed, because the freescout-role changes the permissions and then the content inside the git-directory has changed